wire/wire.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+package basictracer_go.wire;
+option go_package = "wire";
+
+message TracerState {
+  sfixed64 trace_id = 1;
+  sfixed64 span_id = 2;
+  bool    sampled = 3;
```

Comment:
Part of me wants to make this an int32 and use the low bit to represent the sampled bool proper... In order to draw statistical conclusions from sampled trace data it is helpful to record and propagate the effective sampling rate. It's also helpful to record if the trace sampling was "forced" (a la `SAMPLING_PRIORITY` here: http://opentracing.io/use-cases/#setting-sampling-priority-before-the-trace-starts) as a bit here. Oh well.

Having written down the above, I'm fine with leaving this as a bool for now, too.